### PR TITLE
[UR] Update UR to 612a263613b235b0257fcaf2f128fc61b06e0c24

### PR DESCRIPTION
**Pull request by fabiomestre**

### CODE REVIEW

File: sycl/plugins/unified_runtime/CMakeLists.txt [hunk 1]
```diff
@@ -53,14 +53,14 @@ endif()
...
+  # Merge:  20d4f49  9a13af
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Fri Oct 27 16:36:23 2023 +0100
-  #     Merge pull request #961 from hdelan/change-unions-to-stdvariant
-  #     [HIP][CUDA] Change unions in ur_mem_handle_t_ to stdvariant
-  set(UNIFIED_RUNTIME_TAG 036b9cfcd8fb4be9394449b5406e6402580a63c9)
+  # Date:   Thu Oct 26 10:20:03 2023 +0100
+  #         Merge pull request #965 from Bensuo/ewan/L0_internal_event_fix
+  #         [Command-buffer][L0] Reset sync-point events
+  set(UNIFIED_RUNTIME_TAG 55409e43a3f69a4e67e7f30558d12ba8c198a438)
```
Comment (kbenzie) [inline sycl/plugins/unified_runtime/CMakeLists.txt:63]:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 612a263613b235b0257fcaf2f128fc61b06e0c24
  # Merge: 7db941d 55409e4
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Tue Nov 7 10:21:37 2023 +0000
  #     Merge pull request #1011 from fabiomestre/fabio/fix_opencl_leak
  #     [OpenCL] Fix memory leak
  set(UNIFIED_RUNTIME_TAG 612a263613b235b0257fcaf2f128fc61b06e0c24)
```

**Review by kbenzie — CHANGES_REQUESTED** [review]
Now that https://github.com/oneapi-src/unified-runtime/pull/1011 has merged please pull in the latest sycl branch changes, update the `UNIFIED_RUNTIME_REPO`/`UNIFIED_RUNTIME_TAG` values as below, then make this PR ready for review.

File: sycl/plugins/unified_runtime/CMakeLists.txt (hunk 1 above)
Comment (fabiomestre) [reply sycl/plugins/unified_runtime/CMakeLists.txt]: I updated the tag :+1: . Waiting for CI now

**Review by kbenzie — APPROVED** [review]
UR LGTM

**Comment by kbenzie** [comment]
@dm-vodopyanov is there a PR title format you'd like us to use when creating UR bump PRs?

**Comment by dm-vodopyanov** [comment]
> @dm-vodopyanov is there a PR title format you'd like us to use when creating UR bump PRs?

There is no any specific format, but the correct tag (`[UR]`) should be used, `[OpenCL]` is not correct here as OpenCL-related functionality in intel/llvm wasn't modified, and the description should reflect the purpose of this change - in this case, the update of Unified Runtime.

**Comment by kbenzie** [comment]
Okay, thanks @dm-vodopyanov. What's up wiht the [Jenkins/Precommit](https://cje-fm-owrp-prod03.devtools.intel.com/satg-ce-compiler-pipelines/job/SYCL_CI/job/intel/job/Precommit_Check_User/2335/display/redirect) check, anything we need to do?

**Comment by dm-vodopyanov** [comment]
> Okay, thanks @dm-vodopyanov. What's up wiht the [Jenkins/Precommit](https://cje-fm-owrp-prod03.devtools.intel.com/satg-ce-compiler-pipelines/job/SYCL_CI/job/intel/job/Precommit_Check_User/2335/display/redirect) check, anything we need to do?

Just restarted, and it seems failing check is passing now, but one of the passing checks now failing - from logs, all of that because of CI-specific issues.

**Comment by againull** [comment]
Jenkins/Precommit failure is unrelated and was reported to the tools team.

**Pull request merged**